src/test/scala/org/broadinstitute/dsde/workbench/sam/api/MockUserInfoDirectives.scala

```diff
@@ -35,7 +35,9 @@ trait MockUserInfoDirectives extends UserInfoDirectives {
     }.unsafeToFuture()
   }
 
-  override def requireCreateUser(samRequestContext: SamRequestContext): Directive1[SamUser] = workbenchUser match {
+  override def requireUserAllowInactive(samRequestContext: SamRequestContext): Directive1[SamUser] = requireActiveUser(samRequestContext)
+
+  override def withNewUser(samRequestContext: SamRequestContext): Directive1[SamUser] = workbenchUser match {
```

Comment:
nit
```suggestion
  override def withNewUser(samRequestContext: SamRequestContext): Directive1[SamUser] = samUser match {
```
And the message in the `None` case.